docs/DeveloperGuide.md

```diff
@@ -1156,3 +1225,42 @@ testers are expected to do more *exploratory* testing.
    
    4. Launch the app by running `java -jar NurseyBook.jar` in the console. <br>
       Expected: The GUI should pop up with no entries. The console should give warnings about incorrect data format (due to the removal of the `{` character at the start of the `nurseybook.json` file).
+
+
+
+## **Appendix: Effort**
+
+Assuming the effort taken to create AB3 is rated at 100, we would estimate our project to have taken an effort of ___. Numerically, we have over 18000 lines of code contributed for version 1.4 of NurseyBook, with over 550 test cases to achieve high code coverage. 
```

Comment:
We haven’t fill in the number for this

Reply:
This will be filled in in a future PR!

Reply:
Did the effort part become bolded?

Reply:
Strangely it looks bolded in source code but it looks fine in the actual documentation.